app/helpers/users_helper.rb

```diff
@@ -1,156 +1,153 @@
     else
-      gift_number = user.gift_works.visible_to_registered_user.count(:id, :distinct => true)
+      gift_number = user.gift_works.visible_to_registered_user.count(:id, distinct: true)
     end
-    span_if_current ts("Gifts (%{gift_number})", :gift_number => gift_number.to_s), user_gifts_path(user)
+    span_if_current ts('Gifts (%{gift_number})', gift_number: gift_number.to_s), user_gifts_path(user)
   end
 
-  def authored_items(pseud, work_counts={}, rec_counts={})
+  def authored_items(pseud, work_counts = {}, rec_counts = {})
     visible_works = pseud.respond_to?(:work_count) ? pseud.work_count.to_i : (work_counts[pseud.id] || 0)
     visible_recs = pseud.respond_to?(:rec_count) ? pseud.rec_count.to_i : (rec_counts[pseud.id] || 0)
-    items = (visible_works == 1) ? link_to(visible_works.to_s + " work", user_pseud_works_path(pseud.user, pseud)) : ((visible_works > 1) ? link_to(visible_works.to_s + " works", user_pseud_works_path(pseud.user, pseud)) : "")
-    if (visible_works > 0) && (visible_recs > 0)
-      items += ", "
-    end
+    items = visible_works == 1 ? link_to(visible_works.to_s + ' work', user_pseud_works_path(pseud.user, pseud)) : (visible_works > 1 ? link_to(visible_works.to_s + ' works', user_pseud_works_path(pseud.user, pseud)) : '')
+    items += ', ' if (visible_works.positive?) && (visible_recs.positive?)
     if visible_recs > 0
-      items += (visible_recs == 1) ? link_to(visible_recs.to_s + " rec", user_pseud_bookmarks_path(pseud.user, pseud, :recs_only => true)) : link_to(visible_recs.to_s + " recs", user_pseud_bookmarks_path(pseud.user, pseud, :recs_only => true))
+      items += visible_recs == 1 ? link_to(visible_recs.to_s + ' rec', user_pseud_bookmarks_path(pseud.user, pseud, recs_only: true)) : link_to(visible_recs.to_s + ' recs', user_pseud_bookmarks_path(pseud.user, pseud, recs_only: true))
     end
-    return items.html_safe
-  end
-  
-#  def print_pseud_drafts_link(pseud)
-#    total = pseud.unposted_works.size
-#    link_to_unless_current t('my_drafts', :default =>"Drafts") + " (#{total})", drafts_user_pseud_works_path(@user, pseud)
-#  end
-  
-  def authors_header(collection, what = "People")
+    items.html_safe
+  end
+
+  #  def print_pseud_drafts_link(pseud)
+  #    total = pseud.unposted_works.size
+  #    link_to_unless_current t('my_drafts', :default =>"Drafts") + " (#{total})", drafts_user_pseud_works_path(@user, pseud)
+  #  end
+
+  def authors_header(collection, what = 'People')
     if collection.total_pages < 2
       case collection.size
-        when 0
-          "0 #{what}"
-        when 1
-          "1 #{what.singularize}"
-        else
-          collection.total_entries.to_s + " #{what}"
+      when 0
+        "0 #{what}"
+      when 1
+        "1 #{what.singularize}"
+      else
+        collection.total_entries.to_s + " #{what}"
       end
     else
-      %{ %d - %d of %d }% [
+      %( %d - %d of %d )% [
```

Comment:
Ah-ha! I see now -- the % sign goes with the _square brackets_. Sorry, this was my bad and I told you to remove the wrong space. It should be `%( %d - %d of %d ) %[`. Then we'll be ready to merge.